xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift

```diff
@@ -12,7 +12,7 @@ import DZNEmptyDataSet
 
 class CourseContentTableViewController: UITableViewController {
 
-    var course: Course!
+    var cdCourse: Course!
```

Comment:
I was waiting for your input on this. I changed this in the last commit to make clear that this is a core data object from the background, not one that I got out of a list view controller. But afterwards I noticed that we should basically have this all the time